classes/Attribute.php

```diff
@@ -19,20 +19,26 @@
  * needs please refer to http://www.prestashop.com for more information.
  *
  * @author    PrestaShop SA <contact@prestashop.com>
- * @copyright 2007-2015 PrestaShop SA
+ * @copyright 2007-2016 PrestaShop SA
  * @license   http://opensource.org/licenses/osl-3.0.php Open Software License (OSL 3.0)
  * International Registered Trademark & Property of PrestaShop SA
  */
 
+/**
+ * Class AttributeCore
+ */
 class AttributeCore extends ObjectModel
 {
     /** @var int Group id which attribute belongs */
     public $id_attribute_group;
 
     /** @var string Name */
     public $name;
+    /** @var string $color */
     public $color;
+    /** @var int $position */
     public $position;
+    /** @todo Find type */
     public $default;
```

Comment:
We can decide on the deletion later (I will make a pull request for that, but I want **also** to say "will be removed on 1.7.1" for instance).

Too much variables and functions are deprecated and never removed because we don't say when we are going to remove it: we can improve ourselves :)